hoverxref/_static/js/hoverxref.js_t

```diff
@@ -143,7 +156,11 @@ $(document).ready(function() {
                 helper
             );
         }
-    })
+        {# If we're lazy-tipstering, we've used up the click event, so open the modal too #}
+    }){% if hoverxref_tooltip_lazy %}.tooltipster('open');
```

Comment:
```suggestion
    }){% if hoverxref_tooltip_lazy %}.tooltipster('show');
```

I didn't find "open" in the docs. Isn't it `"show"` instead?

Reply:
```js
   // then immediately open the tooltip of the element named "example"
   $('#example').tooltipster('open');
```
is on this page: https://calebjacob.github.io/tooltipster/#getting-started

I can't find where you found "show".. this is the right library right?

Reply:
Hrm... I remember that I got bitten by this some time ago. I'm reading https://www.heteroclito.fr/modules/tooltipster/ which points to 3.0 and the link you shared points to 4.0

We should double check what's the version we are using in this extension. I don't remember from the top of my head.